src/policy_provider/SamplePolicyProvider.h

```diff
@@ -90,6 +77,8 @@ void BSLP_PolicyPredicate_Deinit(BSLP_PolicyPredicate_t *self);
 bool BSLP_PolicyPredicate_IsMatch(const BSLP_PolicyPredicate_t *self, BSL_PolicyLocation_e location,
                                   BSL_HostEID_t src_eid, BSL_HostEID_t dst_eid);
 
+#define BSLP_POLICY_RULE_DESCRIPTION_MAX_STRLEN 100
```

Comment:
Add a short doc for this define and please explain which other API this affects by linking to this in function docs as `#BSLP_POLICY_RULE_DESCRIPTION_MAX_STRLEN`. Users need to know which text lengths are limited.